Added a note for CRD improvement in Docs (#30431)
kubernetes/website: content/en/docs/concepts/storage/volume-snapshot-classes.md

of a class when first creating VolumeSnapshotClass objects, and the objects cannot
be updated once they are created.

{{< note >}}

> Installation of the CRDs are the responsibility of the Kubernetes distribution. Without the required CRDs present the `VolumeSnapshotClass` fails. 

Reviewer: It's not clear to me what "responsibility of the Kubernetes distribution" means here? Do you mean responsibility of the cluster administrator?

```suggestion
Installation of the CRDs is the responsibility of the Kubernetes distribution. Without the required CRDs present, the `VolumeSnapshotClass` fails. 
```

Author: > It's not clear to me what "responsibility of the Kubernetes distribution" means here? Do you mean responsibility of the cluster administrator?

kubernetes distribution is a collection of kubernetes and related technologies and services that we need to use together for deploy and use containers.

Reviewer 2: "the `VolumeSnapshotClass` fails" -> the creation of a `VolumeSnapshotClass` fails

Author: Change the note as suggested.Thanks

> Installation of the CRDs are the responsibility of the Kubernetes distribution. Without the required CRDs present the `VolumeSnapshotClass` below will not get created.  

Reviewer 3: ```suggestion
Installation of the CRDs are the responsibility of the Kubernetes distribution. Without the required CRDs present the `VolumeSnapshotClass` fails. 
```

https://kubernetes.io/docs/contribute/style/style-guide/#use-present-tense

This comment is non-blocking, as I could be sold on this applying the rule's exception: "Use future or past tense if it is required to convey the correct meaning."

> The CRDs  installations are the responsibility of the Kubernetes distribution.Respective CRD's must be apply for  API object. As the API objects `VolumeSnapshot`,`VolumeSnapshotClass` and `VolumeSnapshotContent` are not the part of Core API.

Reviewer: Taking inspiration from the similar note [here](https://kubernetes.io/docs/concepts/storage/volume-snapshots/#introduction), do you think we can simplify this to something like:

```suggestion
Installation of the CRDs are the responsibility of the Kubernetes distribution. Without the required CRDs present the `VolumeSnapshotClass` below will not get created.  
```

WDYT?

Author: @RinkiyaKeDad yes we could wait from some one from sig-storage to approve, and i have added a commit as you suggested.
Thanks

Author: @RinkiyaKeDad whom could we ping regarding review of this PR from sig/storage
